packages/manifest/src/index.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TeamsAppManifest, IStaticTab, IConfigurableTab } from "@microsoft/teamsfx-api";
+import * as fs from "fs-extra";
+
+export class Manifest {
+  private manifest: TeamsAppManifest;
+
+  constructor(manifest?: TeamsAppManifest) {
+    if (manifest) {
+      this.manifest = manifest;
+    } else {
+      this.manifest = new TeamsAppManifest();
+    }
+  }
+
+  static async initWithAppName(appName: string): Promise<Manifest> {
```

Comment:
Should we return `Manifest` instead of `TeamsAppManifest`?
This `Manifest` class seems to have some nice static functions, but I am not seeing the value in having it returned as what people really care about is the interface definition class (which is `TeamsAppManifest`).